README.md

```diff
@@ -1,25 +1,115 @@
 # glazier
 
-`glazier` is an attempt to provide a common interface to the various
-elements of different platform application frameworks. It is designed to be used
-by [druid], a UI toolkit.
+Glazier is a portable infrastructure layer intended for high quality GUI
+toolkits in Rust. It is based on druid-shell but with some changes, primarily
```

Comment:
I don't think leading with `druid-shell` is really necessary - I think a section at the end talking about this lineage would be more suitable.